bin/git-missing

```diff
@@ -38,4 +50,8 @@ else
     exit 1
 fi
 
-git log "${git_log_args[@]}" "$firstbranch"..."$secondbranch" --format="%m %h %s" --left-right
```

Comment:
I think a single codepath can be preserved if `pathspec` is an indexed array and you add `-- "${pathspec[@]}"` at the end?

Reply:
Spot on, you're right. Simplified in b8488cd.